pkg/services/notifications/webhook.go

```diff
@@ -96,16 +98,25 @@ func (ns *NotificationService) sendWebRequestSync(ctx context.Context, webhook *
 	if webhook.Validation != nil {
 		err := webhook.Validation(body, resp.StatusCode)
 		if err != nil {
-			ns.log.Debug("Webhook failed validation", "url", webhook.Url, "statuscode", resp.Status, "body", string(body))
+			ns.log.Debug("Webhook failed validation", "statuscode", resp.Status, "body", string(body))
```

Comment:
We should log the webhook identifier here (and below) since we don't have the url any more, it probably wouldn't hurt to log the error as well.